.github/workflows/build-and-push-bskyweb-aws.yaml

```diff
@@ -4,7 +4,7 @@ on:
   push:
     branches:
       - main
-      - divy/bskylink
+      - divy/starterpack-meta
```

Comment:
not sure if this is ready to go but just pointing to this if it is.

Reply:
```suggestion
```